docs/modules/ROOT/pages/getting-started/roles_persmissions.adoc

```diff
@@ -1,253 +1,258 @@
 = Permissions and roles
 
-== Getting started 
+As a workspace owner, you can add a collaborator to your workspace with their username and assign them a specific role that determines the actions they can take in that workspace.
 
-When the administrators at {ProductName} create a workspace for you, you automatically have the role of *Admin*. 
+The roles available for a workspace, from least to most access, are the following:
 
-== Types of roles
+* *Contributor*: For users who actively push code to your workspace but who don't need to manage workspace configuration.
 
-Invite other users to collaborate in your workspace by entering their {ProductName} username. You can assign one of the following roles to a collaborator:
+* *Maintainer*: For users who need to manage the workspace without access to sensitive actions like managing security.
 
-* *Contributor*: For users who regularly push to your workspace
-* *Maintainer*: For users who need to manage the workspace but who don't need access to sensitive actions like managing security or deleting a workspace
-* *Admin*: For users who need full access to the workspace and who need to perform sensitive actions like managing security or deleting a workspace
+* *Owner*: For users who need full access to the workspace, including sensitive actions like managing security.
 
-== Managing users 
+NOTE: By default, the role for newly invited collaborators is *Contributor*, but you can change this.
 
-To begin adding, editing, or removing users from a workspace, complete the following steps:
+== Adding a user to your workspace
 
-. In the side bar, select *Workspaces*. 
-. Select the tab *Access*. 
+.Prerequisites
 
-=== Adding a user to a workspace
+* The user you want to add must be logged in to {ProductName}.
```

Comment:
They don't necessarily have to be logged in _right now_ at the _same time_ as you.

```suggestion
* The user you want to add must have a valid login to {ProductName}.
```